src/levanter/models/llama.py

```diff
@@ -226,10 +228,12 @@ def attention_config(self) -> AttentionConfig:
             num_kv_heads=self.num_kv_heads,
             head_dim=self.head_dim,
             use_bias=self.use_bias,
+            use_mup=self.use_mup,
             upcast_attn=self.upcast_attn,
             attn_backend=self.attn_backend,
             flash_attention_block_size=self.flash_attention_block_size,
             rope=self.rope,
+            scaling_factor=1.0 if self.use_mup else None,  # Scaling is done by muP
```

Comment:
seems cleaner to change the default behavior in Attention so that None means mup if use_mup and 1/sqrt(d) otherwise

Reply:
SG